src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -66,6 +66,18 @@
                 "JobHeartbeatTimeoutThreshold": "00:10:00",
                 "JobPollingFrequency": "00:00:10",
                 "QueryDelayIntervalInMilliseconds": 500
+            },
+            "DataConvert": {
+                "Enabled": false,
+                "ContainerRegistryServers": [],
+                "TemplateCollectionOptions": {
+                    "ShortCacheTimeSpan": "00:30:00",
+                    "LongCacheTimeSpan": "01:00:00",
+                    "TemplateCollectionSizeLimitMegabytes": 10
+                },
+                "CacheSizeLimit": 100000000,
+                "ContainerRegistryTokenCacheExpiration": "00:30:00",
```

Comment:
Why are we choosing to make this configurable? Should we not look at the `exp` claim in the token?

Reply:
Thanks. Have changed to use the ```exp``` claim (minus 5 minutes as a buffer) as absolute expiration.